src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -244,12 +239,10 @@ def _merge_new_exception_handler(kwargs, handler):
     first = kwargs.get('exception_handler')
 
     def new_handler(ex):
-        try:
-            handler(ex)
-        except Exception:  # pylint: disable=broad-except
```

Comment:
Is any reason to remove try/except?

Reply:
`handler` previously raises exception. Now it doesn't and exceptions are raised later in `new_handler.

Reply:
`handler` previously raises exception. Now it doesn't and the exception is raised later in `new_handler`. See ⬇ code.